index.html

```diff
@@ -45,6 +45,178 @@ <h1>
+              <li class="tag"><span class="tag_txt">Javascript</span></li>
+              <li class="tag"><span class="tag_txt">HTML</span></li>
+            </ul>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
+
+          </section>
+        </section>
+        <section class="grid_1-item">
+          <!-- <img src="#" alt="" class="img_place_holder"> -->
+          <section class="txt">
+            <p class="project_title">Multi-Post Stories Gain+Glory</p>
+            <ul class="tag_ctn">
+              <li class="tag"><span class="tag_txt">Ruby on Rails</span></li>
+              <li class="tag"><span class="tag_txt">CSS</span></li>
+              <li class="tag"><span class="tag_txt">Javascript</span></li>
+              <li class="tag"><span class="tag_txt">HTML</span></li>
+            </ul>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
+
+          </section>
+        </section>
+        <section class="grid_1-item">
+          <!-- <img src="#" alt="" class="img_place_holder"> -->
+          <section class="txt">
+            <p class="project_title">Multi-Post Stories Gain+Glory</p>
+            <ul class="tag_ctn">
+              <li class="tag"><span class="tag_txt">Ruby on Rails</span></li>
+              <li class="tag"><span class="tag_txt">CSS</span></li>
+              <li class="tag"><span class="tag_txt">Javascript</span></li>
+              <li class="tag"><span class="tag_txt">HTML</span></li>
+            </ul>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
 
+          </section>
+        </section>
+        <section class="grid_1-item">
+          <!-- <img src="#" alt="" class="img_place_holder"> -->
+          <section class="txt">
+            <p class="project_title">Multi-Post Stories Gain+Glory</p>
+            <ul class="tag_ctn">
+              <li class="tag"><span class="tag_txt">Ruby on Rails</span></li>
+              <li class="tag"><span class="tag_txt">CSS</span></li>
+              <li class="tag"><span class="tag_txt">Javascript</span></li>
+              <li class="tag"><span class="tag_txt">HTML</span></li>
+            </ul>
+            <section class="action_2 centered">
+              <button class="see">
+                <span class="see_txt">See Project</span>
+              </button>
+            </section>
+
+          </section>
+        </section>
+    </section>
+  </section>
+
+  <section class="about_me">
+    <section class="head">
+      <img src="images/Mask Group.png" alt="Logo decoratid n 43">
```

Comment:
- [ ] The image you have used here is a background image. I recommend you not to use `<img>` tag of HTML background image as it is not a good practice. Kindly use `background-image` property of CSS instead of  `<img>` tag of HTML.

Reply:
Ok done